src/components/header/ProfileThreadJankOverview.js

```diff
@@ -2,6 +2,8 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+// @flow
+
```

Comment:
I've been shooting for >95% flow coverage on my own personal commits. I have nuclide installed with Atom, and it shows flow coverage inline with my code. We also have a flow coverage report, but we haven't really documented it very well. For this first pass I'm fine with just getting flow passing with a `// @flow` declaration, and then divvying up the work to get really good coverage. I haven't loaded this up locally to see what's the actual coverage here.